### ISSUE TITLE
Documentation on old 2.5 develop branch failing - unsupported dependencies :cold_sweat: 

### ISSUE THREAD

**Issue by daquinteroflex** [flexcompute/tidy3d#1399]
Important note: this applies to the develop docs before we actually upgraded the docs. Turns out this problem is caused by sphinx being superseded see the traceback from the develop documentation build from readthe docs:

```
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/flexcompute-tidy3ddocumentation/envs/develop/lib/python3.10/site-packages/sphinx/registry.py", line 438, in load_extension
    metadata = setup(app)
  File "/home/docs/checkouts/readthedocs.org/user_builds/flexcompute-tidy3ddocumentation/envs/develop/lib/python3.10/site-packages/sphinxcontrib/applehelp/__init__.py", line 230, in setup
    app.require_sphinx('5.0')
  File "/home/docs/checkouts/readthedocs.org/user_builds/flexcompute-tidy3ddocumentation/envs/develop/lib/python3.10/site-packages/sphinx/application.py", line 393, in require_sphinx
    raise VersionRequirementError(version)
sphinx.errors.VersionRequirementError: 5.0

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/flexcompute-tidy3ddocumentation/envs/develop/lib/python3.10/site-packages/sphinx/cmd/build.py", line 272, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "/home/docs/checkouts/readthedocs.org/user_builds/flexcompute-tidy3ddocumentation/envs/develop/lib/python3.10/site-packages/sphinx/application.py", line 219, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/flexcompute-tidy3ddocumentation/envs/develop/lib/python3.10/site-packages/sphinx/application.py", line 380, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/flexcompute-tidy3ddocumentation/envs/develop/lib/python3.10/site-packages/sphinx/registry.py", line 441, in load_extension
    raise VersionRequirementError(
sphinx.errors.VersionRequirementError: The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version.

Sphinx version error:
The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version.
```

Solution: Move develop into the latest 2.6 version as soon as possible.

Maybe of interest @tylerflex

**Comment by tylerflex** [comment]
hm, so basically this means that our old docs setup was going to fail anyway and need upgrading? so we did it at the perfect time I guess? :). Maybe we just move ahead with 2.6 unless there's a need for another 2.5 patch?

**Comment by daquinteroflex** [comment]
Haha when I saw the traceback I thought exactly that but didn't want to say it.

Good thing we had the docs ready for upgrade!

**Comment by tylerflex** [comment]
can we close this?

**Comment by daquinteroflex** [comment]
Will be solved in 2.6 release
